ckanext/canada/plugins.py

```diff
@@ -68,6 +68,7 @@ class CanadaSecurityPlugin(CkanSecurityPlugin):
     p.implements(p.IResourceController, inherit=True)
     p.implements(p.IValidators, inherit=True)
     p.implements(p.IConfigurer)
+    p.implements(p.IAuthenticator)
```

Comment:
@RabiaSajjad if you add `, inherit=True` to the `IAuthenticator` plugin you will not need to define the `logout` and `abort` methods here.